src/commands/study.rs

```diff
@@ -240,6 +240,58 @@ pub async fn rm(ctx: &Context, msg: &Message, args: Args) -> CommandResult {
     Ok(())
 }
 
+#[command]
+#[description("Move e renomeia salas da cadeira especificada.")]
+#[usage("cadeira ano_novo semestre_novo [NOME_NOVO]")]
+#[min_args(3)]
+#[required_permissions(ADMINISTRATOR)]
+pub async fn mv(ctx: &Context, msg: &Message, args: Args) -> CommandResult {
+    let trash = ctx.data.read().await;
+    let mut roles = get!(> trash, MiEI, write);
+    let mut args = args.raw();
+    let course = args.next();
+    let new_year = args.next();
+    let new_semester = args.next();
+    if let (Some(c), Some(y), Some(s), Some(g)) = (course, new_year, new_semester, msg.guild_id) {
+        let new_name = args.next().filter(|&n| !n.eq_ignore_ascii_case(c));
+        if let Ok(n) = roles.move_course(c, y, s, new_name, ctx, g).await {
+            msg.channel_id
+                .say(&ctx.http, format!("Cadeira movida: {} -> {}", c, n))
+                .await?;
+        } else {
+            msg.channel_id
+                .say(&ctx.http, "Não foram movidas cadeiras")
+                .await?;
+        }
```

Comment:
You should log the error otherwise it's lost forever and we can't troubleshoot.

Also probably a good idea to add it to the message you're sending to the channel

Reply:
I'm not sure how to send the message because errors in `channels.rs` are written in English but messages to channels should be in Portuguese.

Should I just send:
> Não foram movidas cadeiras:
> *Error message in English*

Reply:
Since these are admin commands I think it's okay. We'll cleanup errors latter, the whole bot needs a better error handling setup. I'd rather have easy to access logs.